Dockerfile-73

```diff
@@ -138,10 +136,7 @@ RUN cp /etc/php/7.0/mods-available/* $CONF_PHPMODS && \
     # Run standard set of tweaks to ensure runs performant, reliably, and consistent between variants
     /bin/bash -e /prep-php.sh && \
     # Add new 7.3+ conf
-    sed -i "s/;decorate_workers_output.*/decorate_workers_output = no/" $CONF_FPMPOOL && \
-    sed -i "s/;log_limit.*/log_limit = \${PHP_FPM_LOG_LIMIT}/" $CONF_PHPFPM && \
```

Comment:
environment variables not valid in `php-fpm.conf` file, moved to startup (run.d) script replacement.